### PR TITLE
Fix a url of the homebrew formula and update copyright year to 2020

### PR DESCRIPTION
**Pull request by gfx**
Just fixed a URL in documents, but also it has `make doc` results, which includes updating copyright years.

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -1,2 +1,3 @@
 /fuzz/http1-corpus/* binary
 /fuzz/http2-corpus/* binary
+/doc/* linguist-generated=true
```
Comment (gfx) [inline .gitattributes]: This is a trick to collapse diffs by default on GitHub.com, because auto-generated files do not matter.

Comment (kazuho) [reply .gitattributes]: Nice hack! Would you mind limiting those to files with certain suffixes? I think we some files (e.g., doc/assets/style.css) are not automatically generated.

Comment (gfx) [reply .gitattributes]: I see! How about 857a5916b96fbf48a749adf3d687dd26941f3dc4?

Comment (kazuho) [reply .gitattributes]: Maybe the mangpages to? I think you might want to take a look at `misc/doc.mk`, and track from the build rule of "all" to find out the files being generated.

Comment (gfx) [reply .gitattributes]: I see.

So I've just rollbacked `.gitattriubutes` in this PR, and I'd like to handle it in another branch in order to maintain docs more easily.

**Comment by kazuho** [comment]
@gfx Would you mind changing the title of this PR to match what it is now, as well as reverting the changes that are made to the automatically generated files?

Once that are being done, I think we can merge this PR, at the same time regenerating the docs in the merge commit.

**Comment by gfx** [comment]
So I should not run `make doc` on updating docs. Okay!

Done!

**Comment by kazuho** [comment]
Thank you for the changes. Merged in the commit above.

**Pull request closed without merging**